modules/RTC/CodecSelection.js

```diff
@@ -124,7 +143,7 @@ export class CodecSelection {
         }
         const currentCodecOrder = session.peerconnection.getConfiguredVideoCodecs();
         const localPreferredCodecOrder = session === this.conference.jvbJingleSession
-            ? this.codecPreferenceOrder.jvb
+            ? this._maybeFilterJvbCodecs()
             : this.codecPreferenceOrder.p2p;
```

Comment:
Why don't we apply the same filtering logic for P2P?

Reply:
P2P is disabled whenever E2EE is enabled. Should we even enable E2EE for P2P?

Reply:
Oh I forgot about that... we should, disabling it was a stop-gap measure.

I can't recall what the problem was though.